From 0da66a61971a4e8b167f2c4d2194e468e31a0aaf Mon Sep 17 00:00:00 2001
From: "Wulf C. Krueger" <philantrop@exherbo.org>
Date: Wed, 26 Jun 2013 17:39:10 +0200
Subject: [PATCH] Use the unrar4 executable

ark is incompatible with unrar >= 5 so I brought back unrar-4.2.4 that
installs only unrar4.
---
 plugins/clirarplugin/cliplugin.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/clirarplugin/cliplugin.cpp b/plugins/clirarplugin/cliplugin.cpp
index db4e416..b7a0064 100644
--- a/plugins/clirarplugin/cliplugin.cpp
+++ b/plugins/clirarplugin/cliplugin.cpp
@@ -64,7 +64,7 @@ ParameterList CliPlugin::parameterList() const
 
     if (p.isEmpty()) {
         p[CaptureProgress] = true;
-        p[ListProgram] = p[ExtractProgram] = QStringList() << QLatin1String( "unrar" );
+        p[ListProgram] = p[ExtractProgram] = QStringList() << QLatin1String( "unrar4" );
         p[DeleteProgram] = p[AddProgram] = QStringList() << QLatin1String( "rar" );
 
         p[ListArgs] = QStringList() << QLatin1String( "vt" ) << QLatin1String( "-c-" ) << QLatin1String( "-v" ) << QLatin1String( "$Archive" );
-- 
1.8.3.1

